ehsm_kms_service/constant.js

```diff
@@ -6,6 +6,7 @@ const CMK_EFFECTIVE_DURATION = 1 * 365 * 24 * 60 * 60 * 1000
 const CMK_LOOP_CLEAR_TIME = 24 * 60 * 60 * 1000
 const CMK_EXPIRE_TIME_EXPAND = 10 * 24 * 60 * 60 * 1000
 const CMK_LOOP_CLEAR_EXECUTION_TIME = 3
+const EHSM_SERVER_VERSION = "v0.1.1"

```

Comment:
how to align it with the version_string defined in the buildenv.mk?

Reply:
it seems that it's better to add another NAPI_GetVersion(), to get the version in the kms-core component, otherwise it's not easy to align them.

Reply:
ok, It will be realized.

Reply:
remove it

Reply:
It fixed.